app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+LOCATION = "Paris"
```

Comment:
The location is hardcoded as 'Paris'. It would be much more flexible to read it from an environment variable, similar to how you're reading the API key. This way, you can easily change the location without modifying the code.